### PR TITLE
Do not select countries repeatedly in a given play round

### LINKED ISSUE #10: Avoid repeated countries in the same playing round
Currently, there is a risk of being asked for the capital of the same country more than once during a round. This is not ideal, as if you know the answer you will be rewarded twice and if you don't you will be disadvantaged! 

There are multiple possible solutions here. For example, you could ask the players how many rounds they would like to play before the game starts, then, generate the list of countries you will be using, check if there are any repeated ones, and if yes, re-generate the list. Another alternative is to reflect more on how the choice of country is being made by `random.choice( )` and whether we would want to ensure that the countries are being sampled and not replaced in your dictionary when doing so (i.e. sampling without replacement).

### PR DESCRIPTION
**Pull request by SoniaBarasa**
* Resolves #10 

This pull request will change the code to ensure the countries once selected cannot be repeated during a given play round. Previously, the countries could be randomly chosen without replacement hence could be  easily repeated.

### CODE REVIEW

File: gameofgame.py [hunk 1]
```diff
@@ -42,9 +42,14 @@ def check_guess(guess, country):
 
 
 # Main game loop
+##listing available countries
+available_countries =list(countries_capitals.keys())
 while True:
-    # Choose a random country
-    country = choose_country()
+    if not available_countries:
+        print("All countries have been played!")
```
Comment (ahernank) [inline gameofgame.py:49]:
```suggestion
    if not available_countries:
        print("All countries have been played!")
        break
```

File: gameofgame.py [hunk 2]
```diff
@@ -42,9 +42,14 @@ def check_guess(guess, country):
...
+    # Choose a random country with replacement
+    country = random.sample(available_countries, 1) [0]
```
Comment (ahernank) [inline gameofgame.py:51]:
```suggestion
    country = random.sample(available_countries, 1)[0]
```

**Review by ahernank — COMMENTED** [review]
Thanks @SoniaBarasa! Adding some small suggestions.

**Review by SoniaBarasa — COMMENTED** [review]
@ahernank @alimanfoo thank you very much for your review. I have incorporated the suggestions in the code.

**Review by ahernank — APPROVED** [review]
Thanks @SoniaBarasa! Looks good to me.

**Comment by SoniaBarasa** [comment]
Thank you very much @ahernank for the review, I will merge this pull request as you have mentioned.

**Pull request merged**